packages/odf/components/s3-browser/bucket-details/BucketDetails.tsx

```diff
@@ -166,16 +169,23 @@ const BucketDetailsOverview: React.FC<{}> = ({}) => {
 };
 
 type BucketDetailsProps = {
-  obj: K8sResourceKind;
+  obj: {
+    refresh: boolean;
+    resource?: K8sResourceCommon;
+  };
 };
 
-export const BucketDetails: React.FC<BucketDetailsProps> = ({ obj }) => {
+export const BucketDetails: React.FC<BucketDetailsProps> = ({
+  obj: { resource, refresh },
+}) => {
   const { t } = useCustomTranslation();
 
-  return (
+  return refresh && resource ? (
     <>
       <BucketDetailsOverview />
-      {obj && <OBDetails obj={obj} ownerLabel={t('Owner References')} />}
+      {<OBDetails obj={resource} ownerLabel={t('Owner References')} />}
```

Comment:
`BucketDetailsOverview` does not depends on CR (resource)...
```suggestion
      {resource && <OBDetails obj={resource} ownerLabel={t('Owner References')} />}
```